hashin.py

```diff
@@ -55,7 +55,7 @@ def _download(url, binary=False):
     return r.read().decode(encoding)
 
 
-def run(spec, file, algorithm, python_versions=None, verbose=False):
+def run(spec, file, algorithm, python_versions=[], verbose=False):
```

Comment:
Generally a bad idea to use a mutable type as a default in a function signature though. Should keep it `None` then `python_versions = python_versions or []` as the first line. Or, really, there's no need to change this line, only line 78.